paasta_tools/long_running_service_tools.py

```diff
@@ -87,6 +98,9 @@ def is_in_smartstack(self) -> bool:
         else:
             return False
 
+    def get_healthcheck_cmd(self) -> Optional[str]:
```

Comment:
This isn't right. We don't allow people to specify a healthcheck_cmd in their smartstack config file.

I mean, I guess we could?

Reply:
oh, you're right, my mistake